src/app/CMakeLists.txt

```diff
@@ -161,10 +161,12 @@ endif (GUI AND WIN32)
 target_link_libraries(${QBT_TARGET_NAME} ${QBT_TARGET_LIBRARIES} QtSingleApplication::QtSingleApplication)
 
 if (APPLE)
-    set(qbt_BUNDLE_NAME "${CMAKE_PROJECT_NAME}")
+    set(qbt_BUNDLE_NAME "${QBT_TARGET_NAME}")
+    execute_process(COMMAND bash -c "sed -e \"s,@EXECUTABLE@,${qbt_BUNDLE_NAME},g\" ${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist > ${qBittorrent_BINARY_DIR}/Info.plist.in")
```

Comment:
Thanks for reviewing.

When I build -nox version, I need to modify `qbittorrent` to `qbittorrent-nox` in 2 fields in `dist/mac/Info.plist`.
Qmake takes care of it if I use the `@EXECUTABLE@` variable instead of hard coding.
Cmake has a different syntax for this like below so I have to explicitly modify them by `sed`.
I couldn't figure out where is the good place to put the modified Info.plist (temporary file).
You must be a cmake expert. Any advice?

```
	<key>CFBundleExecutable</key>
	<string>${MACOSX_BUNDLE_EXECUTABLE_NAME}</string>
	<key>CFBundleName</key>
	<string>${MACOSX_BUNDLE_BUNDLE_NAME}</string>
```